apps/drec-api/src/pods/device/dto/filter.dto.ts

```diff
@@ -0,0 +1,65 @@
+import { IsOptional } from 'class-validator';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import {
+  Installation,
+  OffTaker,
+  Sector,
+  StandardCompliance,
+} from '../../../utils/eums';
+
+export class FilterDTO {
+  @IsOptional()
+  @ApiPropertyOptional({ type: String, description: 'Fuel Code' })
+  fuel_code: string;
```

Comment:
just wondering, why we are using sometimes camelCase and sometimes snake_case?

Reply:
Yeah, I don`t like it either. I used snake_case for the entity columns (same as SBE) and then for the same column filters, I used the same, but for others, out of habit I used camelCase. I should use snake_case for this everywhere to be consistent. 
Thanks

Reply:
You shold use camelCase everywhere in code IMHO, because that's our code style.

If database is using snakecase for whatever reason (I know the reason) it should not propagate to the rest of the code.
But postgres can use camelCase, typeorm should automatically build correct queries.

So TBH I have no idea what's the things with snake case at all